perception/multi_object_tracker/src/tracker/model/tracker_base.cpp

```diff
@@ -54,6 +54,60 @@ bool Tracker::updateWithoutMeasurement()
   return true;
 }
 
+void Tracker::updateClassification(
+  const std::vector<autoware_auto_perception_msgs::msg::ObjectClassification> & classification)
+{
+  // classification algorithm:
+  // 0. Remove the class with probability < 0.005
+  // 1. Decay all existing probability
+  // 2. Update the matched classification probability with a gain
+  // 3. If the label is not found, add it to the classification list
+  // 4. Normalize
+
+  // Gain and decay
+  const double gain = 0.05;
+  const double decay = 1.0 - gain;
+
+  // If the probability is less than 0.005, remove the class
```

Comment:
How about put this after decay? or in the last part after normalization like pruning process. (Then we need normalization twice.)